recipe/meta.yaml

```diff
@@ -31,10 +29,10 @@ outputs:
       run:
         - python
         - packaging
-        - pandas >=1.0.0
-        - pyproj >=2.6.1.post1
-        - shapely >=1.7
-        - fiona >=1.8
+        - pandas >=1.1.0
+        - pyproj >=3.0.1
+        - shapely >=1.7.1
+        - fiona >=1.8.19
```

Comment:
In the conda-forge recipe, we moved fiona from `geopandas-base` to `geopandas`

Reply:
Hm, but then I suspect that `pip check` will complain about missing required dependency https://github.com/geopandas/geopandas/blob/d5add48e966bb4e711d2399721df88349a358904/pyproject.toml#L30

Reply:
I found that `import fiona` takes place only here in the try/except block https://github.com/geopandas/geopandas/blob/32825981fdd5518ea41352507d0982401a870317/geopandas/tools/_show_versions.py#L61 so moving it from `geopandas-base` to `geopandas` makes sense.

Reply:
> Hm, but then I suspect that pip check will complain about missing required dependency

That's indeed the case, so in conda-forge we don't have a `pip check`